root/documentation/vep.conf

```diff
@@ -154,6 +154,11 @@
         description=Determines where in the secondary structure of a miRNA a variant falls (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/miRNA.pm">plugin details</a>)
         default=0
       </miRNA>
+      <LoF>
+        type=Boolean
+        description=LOFTEE identifies LoF (loss-of-function) variation (<a target="_blank" href="https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm">plugin details</a>)
```

Comment:
Would it be better to link this to the loftee README.md rather than the raw Perl code?